fractal_server/main.py

```diff
@@ -138,6 +139,8 @@ async def _create_first_user(
                     )
                     if slurm_user:
                         kwargs["slurm_user"] = slurm_user
+                    if slurm_accounts:
```

Comment:
Let's not use this pattern (`if something` instead of `if something is not None`).